pkg/sign/signer.go

```diff
@@ -22,16 +22,6 @@ import (
 const bundleV03MediaType = "application/vnd.dev.sigstore.bundle.v0.3+json"
 
 func Bundle(content Content, keypair Keypair, fulcio *Fulcio, idToken string, timestampAuthority *TimestampAuthority) (*protobundle.Bundle, error) {
```

Comment:
Can we document what is and isn't required, and also err out when the required params aren't set? content and keypair are always, idtoken is required only when fulcio is set.

Reply:
I promise that I will do this in a sec when I post Rekor support.